Horizontal divider between main buttons and tags

TurboProgramming opened the pull request:
**Changes proposed in this pull request:**
Addition of horizontal separator between main buttons and tags.

**Reviewers should focus on:**
[sideNav.less](https://github.com/Flarum/core/blob/1f257d22a441e9101d4d47bb900be872d5a38943/less/common/sideNav.less) --> line 68 - 116

**Supports:**
- Dark Mode 🎨

**Screenshot**
![screenshot_2018-07-17 appybuilder community](https://user-images.githubusercontent.com/17576065/42838827-7ceb4b50-8a0a-11e8-93e3-acb71185f5ad.png)

datitisev commented:
Why are you modifying the styles for the composer separator if this is only supposed to affect the navbar ?
Also, does this work when the tags extension is _not_ enabled?

franzliedke commented on less/common/sideNav.less:
```diff
@@ -65,6 +65,56 @@
+    margin:0
+    }
+    
+    .TagsPage .sideNav .Dropdown-separator {
```
Styling (or code) related to a core extension should never be part of flarum/core.

franzliedke commented:
That said, I believe Toby styled things intentionally. Unless you can provide a very good reason why we should change this, I am pretty confident this won't get merged.

Closing for now.

TurboProgramming commented:
A space could be a great way to separate stuff but this divider really looked great.

With inspector, it doesn't work, but I believe it's an issue with less and vanilla CSS again.

TurboProgramming commented:
@franzliedke Can you reopen this PR as I will be modifying some stuff?

Maybe we'll decide to accept this change, I can't modify it for the PR.

TurboProgramming commented:
> Why are you modifying the styles for the composer separator if this is only supposed to affect the navbar?

I removed the lines, I added them by mistake. Sorry 😞 

> Also, does this work when the tags extension is not enabled?

Probably no, if this PR gets opened again, I can make modify some stuff to make it work only if the tags extension is enabled.

(@datitisev)

franzliedke commented:
We can reopen it in case Toby likes the idea.

For now, let's save some time for all of us.
